components/slot-fill/README.md

```diff
@@ -47,3 +47,5 @@ You can either use the Fill component directly, or a wrapper component type as i
 The `SlotFillProvider` component does not accept any props.
 
 Both `Slot` and `Fill` accept a `name` string prop, where a `Slot` with a given `name` will render the `children` of any associated `Fill`s.
+
+`Slot` also accepts a `bubble` event which changes the rendering behavior. If we want to bubble events to the Fills parent component, we need to pass this prop to the correspondig `Slot`.
```

Comment:
The naming and description of `bubble` doesn't really clarify its intended effect here. In either case, events bubble, it just varies on which ancestor hierarchy those events bubble through (virtual vs. DOM ancestry). Personally I don't think an implementer needs to care that the render behavior is different at all, and we should focus on how they want events to be handled. Maybe if this is an opt-in we should clarify that it is opting into bubble through the virtual hierarchy:

- `bubblesVirtual`
- `bubblesVirtually`
- `bubblesVTree`
- `bubblesDOM={ false }`
- `bubblesNative={ false }`
- `bubblesNatively={ false }`
- `nativeBubbling={ false }`